### PR TITLE
feat: i18n for aggregate data exchange job type [DHIS2-13552]

### PR DESCRIPTION
**Pull request by jbee**

### CODE REVIEW

**Comment by jbee** [comment]
@ismay Yes. Just that the parameter is a **list** of aggregate data exchange objects.

**Review by HendrikThePendric — CHANGES_REQUESTED** [review]
This is not working as expected. After selecting the new job-type the app crashes. See [this Slack thread](https://dhis2.slack.com/archives/CPGUFVC56/p1660556381100509) for more info.

**Comment by DNR800** [comment]
Have put a fix in place for the app crash when trying to add a Exchange aggregate data job

Visible state below

**Loading State for the add form for the Exchange aggregate data job**
loading-state

**Fetching Data Ids Error State for the add form for the Exchange aggregate data job**
error-state

**Working add form for the Exchange aggregate data job**
displaying-transfer

File: src/components/FormFields/AggregatedDataExchangeField.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    FieldGroup,
+    Help,
+    InputFieldFF,
+    NoticeBox,
+    ReactFinalForm,
+} from '@dhis2/ui'
+import React from 'react'
+import { PropTypes } from '@dhis2/prop-types'
+import { useDataQuery } from '@dhis2/app-runtime'
+import styles from './DataIntegrityChecksField.module.css'
```
[the remark this answers is not in the document]
Comment (DNR800) [reply src/components/FormFields/AggregatedDataExchangeField.js]: just commit a change for this creating a shared css module

File: src/components/FormFields/AggregatedDataExchangeField.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            </NoticeBox>
+        )
+    }
+
+    const options = data.dataExchangeIds.aggregateDataExchanges
+        ? data.dataExchangeIds.aggregateDataExchanges.map((exchangeIds) => ({
+              label: exchangeIds.displayName,
+              value: exchangeIds.id,
+          }))
+        : []
```
Comment (Birkbjo) [inline src/components/FormFields/AggregatedDataExchangeField.js:76]: Very minor, but could simplify to this:

```suggestion
     const options = data.dataExchangeIds?.aggregateDataExchanges.map((exchangeIds) => ({
              label: exchangeIds.displayName,
              value: exchangeIds.id,
          }))
        ?? []
```

**Review by Birkbjo — APPROVED** [review]
LGTM!

File: src/components/FormFields/AggregatedDataExchangeField.js (hunk 2 above)
Comment (DNR800) [reply src/components/FormFields/AggregatedDataExchangeField.js]: @Birkbjo Thanks, have added in that simplification you suggest

**Review by HendrikThePendric — APPROVED** [review]
Sorry for blocking this PR. Looks great, thanks a lot @DNR800

**Pull request merged**